Sources/Error.swift

```diff
@@ -12,12 +12,21 @@ protocol ANSIConvertible : Error, CustomStringConvertible {
 
 extension ANSIConvertible {
   func print() {
-    if isatty(fileno(stderr)) != 0 {
+    // Check if we are in any term env and the output is a tty.
+    let termType = getEnvValue("TERM")
+    if let t = termType, t.lowercased() != "dumb" && isatty(fileno(stdout)) != 0 {
       fputs("\(ansiDescription)\n", stderr)
     } else {
       fputs("\(description)\n", stderr)
     }
   }
+
+  private func getEnvValue(_ key: String) -> String? {
+    guard let value = getenv(key) else {
+      return nil
+    }
+    return String(cString: value)
+  }
```

Comment:
Right, so add the method back in again? It is clear what it does, and nicely wraps around unsafe pointer stuff.